public/player.js

```diff
@@ -35,32 +35,42 @@
     YT.ready(onYouTubeIframeAPIReady);
   };
 
-  function toggleHighlight(x) {
-    x.classList.toggle("highlighted-option")
-    setTimeout(function() {
-      x.classList.remove("highlighted-option")
-    }, 3000)
+  var funcCount = (function () {
```

Comment:
What are we counting?

this name (`funcCount`) implies we are counting functions, should this be called `questionsAnsweredCounter` or something?